package.json

```diff
@@ -5,7 +5,8 @@
   "jest": {
     "coverageReporters": [
       "json",
```

Comment:
If you're using lcov do you also need json?

Reply:
Good catch, I guess not. 9b6063c